testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/RunAsPrincipalTestCase.java

```diff
@@ -218,4 +232,40 @@ private Throwable checkEjbException(Throwable ex) {
         }
         return ex;
     }
+
+    private boolean logContainsWARN() {
+        String jbossHome = System.getProperty("jboss.inst");
+        String logPath = String.format("%s%sstandalone%slog%sserver.log", jbossHome,
+                (jbossHome.endsWith(File.separator) || jbossHome.endsWith("/")) ? "" : File.separator,
+                File.separator, File.separator);
+        logPath = logPath.replace('/', File.separatorChar);
+        BufferedReader fileReader = null;
+        try {
+            fileReader = Files.newBufferedReader(Paths.get(logPath)); // UTF8 is used by default
+        } catch (MalformedInputException e1) {
+            // some windows machines could accept only StandardCharsets.ISO_8859_1 encoding
+            try {
+                fileReader = Files.newBufferedReader(Paths.get(logPath), StandardCharsets.ISO_8859_1);
+            } catch (IOException e4) {
+                throw new RuntimeException("Server logs has not standard Charsets (UTF8 or ISO_8859_1)");
+            }
+        } catch (IOException e) {
+            // server.log file is not created, it is the same as server.log is empty
+        }
+
+        String line = "";
+        boolean found = false;
+        try {
+            while ((line = fileReader.readLine()) != null) {
+                if(line.contains("@RunAs annotation is required when using @RunAsPrincipal")) {
```

Comment:
We need to use the message id, `WFLYEJB00510`. Once we have localization done the output may be in a different language.